types/src/chainspec/vm_config/opcode_costs.rs

```diff
@@ -531,6 +582,48 @@ impl FromBytes for OpcodeCosts {
     }
 }
 
+impl Zero for OpcodeCosts {
+    fn zero() -> Self {
+        Self {
+            bit: 0,
+            add: 0,
+            mul: 0,
+            div: 0,
+            load: 0,
+            store: 0,
+            op_const: 0,
+            local: 0,
+            global: 0,
+            integer_comparison: 0,
+            conversion: 0,
+            unreachable: 0,
+            nop: 0,
+            current_memory: 0,
+            grow_memory: 0,
+            control_flow: ControlFlowCosts::zero(),
+        }
+    }
+
+    fn is_zero(&self) -> bool {
+        self.bit.is_zero()
```

Comment:
Can you use `let OpcodeCosts { bit, add, .. } = self` etc just so we wouldn't accidentally forget to update this if it grows new fields?

Reply:
Changed in https://github.com/casper-network/casper-node/commit/53483e16590b67de9be9be4d9ba4e6b64d1f22ee